playground/bicep/BicepSample.ApiService/Program.cs

```diff
@@ -179,7 +179,7 @@
     var message = await queue.ReceiveMessageAsync(cancellationToken: cancellationToken);
     if (message != null)
     {
-        entries.Add(message.Value.Body.ToObjectFromJson<Entry>());
+        entries.Add(message.Value.Body.ToObjectFromJson<Entry>()!);
```

Comment:
Was this related?

Reply:
Yes - because your change lifted to the new Azure.Core 1.44 version, which brings in a new System.Memory.Data (v6). The old version didn't have nullable annotations, and this new version does.